### ISSUE TITLE
Machine Controller Covers occasionally get locked and do not respect the current signal

### ISSUE THREAD

**Issue by windyknight** [GregTechCEu/GregTech#2357]
### GregTech CEu Version

2.7.4

### Modpack Used

Nomifactory CEu 1.7 alpha 2a

### Environment

Singleplayer

### New Worlds

Unsure

### Cross-Mod Interaction

Yes

### Expected Behavior

Machines controlled by AE2 level emitters always respect the signal given by the emitter, activating and deactivating as usual.

### Actual Behavior

Occasionally, a controller will lock up and stay inactive despite the emitter signal signifying that it should be active. This causes important resources in a chain to drain out and halt production entirely. Toggling the emitter's threshold behavior back and forth, which gives a momentary signal toggle, jolts the controller out of the paralyzed state.

### Steps to Reproduce

1. Set up a production line using singleblock machines, using AE2 interfaces to push and pull requisite items from. Use drawers with void upgrades as the storage area for the resources in the chain. In my world, this is my Pulsating Polymer Clay setup.
2. Control the production of all resources using AE2 level emitters set to emit above specific quantities of the machine's product. Machine controllers left on default settings (signal present means to deactivate a machine). In my world, this quantity is 2000 for everything except for clay blocks, which are set to 1000.
3. Wait. This might take a while.
4. Notice at some point that production has stopped. There will be at least one machine whose controller cover was locked to inactive even though there is no redstone signal, as the amount of the product fell below the threshold.
5. Open the level emitter's UI and switch the signal behavior twice (to emit below threshold and back to emit above threshold). The signal change returns the cover's behavior to normal.

### Additional Information

My suspicion is that because AE2 emitters use a single threshold instead of utilizing a latch behavior, they flicker between their on and off states relatively frequently. This flickering might be what catches the controllers in awkward states.

**Comment by windyknight** [comment]
Video evidence of this occurring with my Multi Smelter's machine controller cover. Occurred about 2 hours or so into my current session.

https://github.com/GregTechCEu/GregTech/assets/9163117/f958fcd4-fa61-4f29-8105-1d27817d824d

**Comment by MatthiasMann** [comment]
Is the level emitter in the same chunk as the EBF controller?

**Comment by windyknight** [comment]
Yes. This whole set of machines is within a chunk. Plus, it and all the surrounding chunks are forceloaded.

**Comment by windyknight** [comment]
Updated to 2.8.6 via the NomiCEU nightly. No change, bug still persists.
